service/resource/kmskeyv1/create.go

```diff
@@ -1,11 +1,67 @@
 package kmskeyv1
 
-import "context"
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/kms"
+	"github.com/giantswarm/aws-operator/service/keyv1"
```

Comment:
Super nitpicky but please separate this import.

Reply:
Sure done :)